### DISCUSSION TITLE
[Feature] Move clock/date during sleep

### DISCUSSION THREAD

**Discussion by I-Would-Like-To-Report-A-Bug-Please** [damongolding/immich-kiosk#154] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great if the clock/date would move during sleep hours. 

This could potentially save some screens from burnout in those specific places when running for long time with similar colours.

**Comment by damongolding** [comment]
Good suggestion!

**Comment by damongolding** [comment]
implemented in 0.13.0
